__tests__/api.test.js

```diff
@@ -70,6 +70,93 @@ describe("/api/articles", () => {
         });
       });
   });
+  describe("GET - queries", () => {
```

Comment:
Apologies - I didn't mention/spot this before but we need to test for what happens (regarding the error handling) if an invalid `sort_by` or `order` query is made. I can see you have taken care of this in other models so please add tests and functionality before merging - no need to resubmit. HINT: these would be _bad requests_